client/.eslintrc

```diff
@@ -2,10 +2,10 @@
   "parser": "babel-eslint",
   "extends": "airbnb",
   "settings": {
-    import/resolver: "webpack"
+    "import/resolver": "webpack"
```

Comment:
You tried removing all of `settings: {...}` right? Not sure if I missed something, but I haven't managed to get it to work on my local and it's failing on #2017.